python/pyspark/ml/classification.py

```diff
@@ -356,6 +356,9 @@ def intercept(self):
         """
         return self._call_java("intercept")
 
+    def __repr__(self):
```

Comment:
Can we put this method to some common class (like `JavaModel`) and add `__repr__` only in classes not extending `JavaModel` ?